frontend/src/components/App/Layout.tsx

```diff
@@ -117,23 +162,27 @@ export default function Layout({}: LayoutProps) {
    */
   const fetchConfig = () => {
     const clusters = store.getState().config.clusters;
+    const statelessClusters = store.getState().config.statelessClusters;
     request('/config', {}, false, false)
       .then((config: Config) => {
         const clustersToConfig: ConfigState['clusters'] = {};
         config?.clusters.forEach((cluster: Cluster) => {
           clustersToConfig[cluster.name] = cluster;
+          if (clustersToConfig[cluster.name].meta_data.source === 'dynamic_cluster') {
```

Comment:
I think this check is not necessary as the /config endpoint doesn't expose any internal clusters. There can be a case where we have dynamic_cluster that is not stateless.